evap/evaluation/models.py

```diff
@@ -1827,6 +1840,12 @@ def is_editor_or_delegate(self):
     def is_responsible_or_contributor_or_delegate(self):
         return self.is_responsible or self.is_contributor or self.is_delegate
 
+    @cached_property
+    def show_startpage_button(self):
+        return [self.is_student, self.is_responsible_or_contributor_or_delegate, self.is_grade_publisher].count(
+            True
+        ) > 1
```

Comment:
Could a user loose their status as for example "student" and then be unable to update their startpage? In addition I think this should also be "participant" instead of student (and maybe the other thing is not a problem anymore then?)

In general, maybe we should have `can_visit_student_index` etc to not have this knowledge duplicated? (Sorry for the general scope bleed :/ )

Reply:
i didn't implement anything like `can_visit_student_index` because this might get even more confusing when thinking about staff mode.